vita3k/gui/src/settings_dialog.cpp

```diff
@@ -334,7 +444,157 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
     } else {
         ImGui::PopStyleColor();
     }
-
+    //Controls
+    ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
```

Comment:
same, create or just use already controller dialoh ecxist for mapping, no içnside setting

Reply:
ok done